components/automate-deployment/pkg/server/automate_config.go

```diff
@@ -280,6 +281,33 @@ func setConfigForRedirectLogs(req *api.PatchAutomateConfigRequest, existingCopy
 		}
 	}
 
+	//Set the config if already not set
+	if req.GetConfig().GetGlobal().GetV1().GetLog().GetRedirectSysLog().GetValue() == true &&
+		existingCopy.GetGlobal().GetV1().GetLog().GetRedirectSysLog().GetValue() == true {
+
+		err := createConfigFileForAutomateSysLog(req.GetConfig().GetGlobal().GetV1().GetLog().GetRedirectLogFilePath().GetValue())
+		if err != nil {
+			return status.Error(codes.Internal, err.Error())
+		}
+
+		if req.GetConfig().GetGlobal().GetV1().GetLog().GetRedirectLogFilePath().GetValue() != existingCopy.GetGlobal().GetV1().GetLog().GetRedirectLogFilePath().GetValue() {
+			err = restartSyslogService()
```

Comment:
if Both the valued are equal then only update log-rotate policies, not when they are unequal. If they are equal i.e unchanged, that means log rotate policies are changed.